src/pivot-table/components/cells/ColumnAdjuster.tsx

```diff
@@ -14,47 +13,41 @@ interface AdjusterProps {
   dataModel: DataModel | undefined;
   isLastColumn: boolean;
 }
-
 /**
  * Component that is placed on top of column border, to resize the columns.
  * When you start dragging, mouse move and mouse up listeners are added.
  * While dragging this components follows the pointer, and on mouse up all column widths are updated.
  */
 const ColumnAdjuster = ({ cellInfo, columnWidth, dataModel, isLastColumn }: AdjusterProps) => {
-  const [internalWidth, setInternalWidth] = useState(columnWidth);
-  const tempWidth = useRef({ initWidth: 0, columnWidth: 0, initX: 0 });
+  const [, setInternalWidth] = useState(columnWidth);
```

Comment:
Maybe rename `setInternalWidth` to `forceRerender` or something like that and just set the value to a new object when we need it to rerender? As I think it can be confussing to keep the column width value here but not actually using it.